metal/mmtl/slicing/launch.py

```diff
@@ -122,8 +127,37 @@ def main(args):
                     )
                 )
 
-    # Initialize and train model
-    model = model_class(tasks, **model_config)
+    if args.model_type == "moe":
+        experts = {}
+        for slice_name in slice_dict[base_task_name]:
+            if slice_name == "BASE":
+                continue
+            task_config.update({"slice_dict": {base_task_name: [slice_name]}})
+            tasks_slice, payloads_slice = create_glue_tasks_payloads(
+                task_names, **task_config
+            )
+            tasks_slice = convert_to_slicing_tasks(tasks_slice)
+            print(tasks_slice)
+            print(payloads_slice)
+
+            # remove the base task labels from the expert payloads.
+            for p in payloads_slice:
+                p.labels_to_tasks.pop(f"{base_task_name}_gold")
+            # remove the slice task labels from the payloads used to train the MoEModel.
+            for p in payloads:
+                p.labels_to_tasks.pop(f"{base_task_name}_slice:{slice_name}:pred")
+
+            model = MetalModel(tasks_slice[1:], verbose=False)

```

Comment:
let's add a comment here about why [1:] removes the base task? or make it more explicit